public/drawio_plugin.js

```diff
@@ -0,0 +1,222 @@
+let iframe = null;
```

Comment:
if we copied this or any other code from another project, please include in a comment at the top of the line: 

1. the Copyright line from the license. typically line 3 if from MIT license. if you read the text, this is a  requirement of the license
2. a link (URL) to the source